exercises/concept/low-power-embedded-game/.meta/design.md

```diff
@@ -21,6 +21,7 @@ Introduce the student to tuples and how to work with them.
 
 ## Resources to refer to
 
+- [Cairo Book - The Tuple Type](https://book.cairo-lang.org/ch02-02-data-types.html#the-tuple-type)
+
 ### Hints
```

Comment:
This file isn't exposed to any students or mentors so we don't need a hints section.